pkg/xds/server/callbacks/dataplane_status_tracker.go

```diff
@@ -101,7 +102,18 @@ func (c *dataplaneStatusTracker) OnStreamClosed(streamID int64) {
 	// trigger final flush
 	state.Close()
 
-	statusTrackerLog.V(1).Info("OnStreamClosed", "streamid", streamID, "subscription", subscription)
+	log := statusTrackerLog.WithValues(
+		"streamID", streamID,
+		"name", state.dataplaneId.Name,
```

Comment:
Why is it called `proxyName` in other places?

Reply:
Thanks, changed.